static/external-player-map.json

```diff
@@ -43,7 +43,7 @@
         "value": "iina",
         "cmdArguments": {
             "defaultExecutable": "iina",
-            "defaultCustomArguments": "--no-stdin",
+            "defaultCustomArguments": ["--no-stdin"],
```

Comment:
Havent tested but since this change is made, I think this line needs to be updated: https://github.com/FreeTubeApp/FreeTube/blob/3f430c0b0dfca3a09e7478fa8cce1e921b72a518/src/renderer/components/external-player-settings/external-player-settings.js#L55